deploy/tools/envs-validator/schema.ts

```diff
@@ -103,12 +102,40 @@ const adButlerConfigSchema = yup
       .required(),
   });
 
+const adCustomConfigSchema = yup
+  .object()
+  .shape({
+    banners: yup
+      .array()
+      .of(
+        yup.object().shape({
+          text: yup.string(),
+          url: yup.string().url().required(),
+          desktop: yup.string().url().required(),
+          mobile: yup.string().url().required(),
+        }),
+      )
+      .required(),
+    interval: yup.number(),
```

Comment:
I would add that it should be a positive integer as well

Reply:
Yes, you're right